massa-execution-exports/src/event_store.rs

```diff
@@ -9,7 +9,7 @@ use std::collections::VecDeque;
 
 /// Store for events emitted by smart contracts
 #[derive(Default, Debug, Clone)]
-pub struct EventStore(VecDeque<SCOutputEvent>);
+pub struct EventStore(pub VecDeque<SCOutputEvent>);
```

Comment:
Why you changed this to public ?

Reply:
To avoid reimplementing iterator functions for `EventStore`, need them to perform a diff with the snapshot in order to deduce events coming from failed executions